src/classes/PageList.php

```diff
@@ -49,10 +54,23 @@ private function find_sibling_pages() {
   }
 
   private function find_child_pages() {
-    $args = array(
-      'parent' => $this->origin,
-      'sort_column' => 'menu_order',
-    );
+    //V: nick-branch
+    /**
+     * Might be missing ID
+     */
+    if (get_post_ancestors( $post )) {
```

Comment:
In the if statement on L61 - L72, what is each branch checking for?

Reply:
if statement checks if $post is the top level with no child. 
Fixed Issue where all side-by-side parent pages would be called and brought into sidebar.
If true - only try to collect childe
Else - this is child. get my parents ID